app/src/main/java/com/isc/hermes/utils/LocationListeningCallback.java

```diff
@@ -0,0 +1,62 @@
+package com.isc.hermes.utils;
+
```

Comment:
Delete the spaces that you do not get to use.

Reply:
@VillalobosGarciaGaby  This recommendation is already satisfied.